ports/openal-soft/portfile.cmake

```
@@ -46,7 +50,7 @@ vcpkg_configure_cmake(
         -DALSOFT_BACKEND_PULSEAUDIO=OFF
         -DALSOFT_BACKEND_COREAUDIO=${ALSOFT_REQUIRE_APPLE}
         -DALSOFT_BACKEND_JACK=OFF
-        -DALSOFT_BACKEND_OPENSL=OFF
+        -DALSOFT_BACKEND_OPENSL=${ALSOFT_REQUIRE_ANDROID}
```

Should this just be:

```suggestion
        -DALSOFT_BACKEND_OPENSL=${VCPKG_TARGET_IS_ANDROID}
```

rather than introducing another variable?

(Genuinely asking if you think so, not necessarily asking you to make that change)

---

I think you are right, but notice that ALSOFT_REQUIRE_LINUX and ALSOFT_REQUIRE_WINDOWS is done in similar manner I just tried base my patch on previous commits.